src/Freya.Core/Builder.fs

```diff
@@ -26,30 +26,30 @@ open Freya.Core.Operators
 
 (* Builder *)
 
-/// Defines a computation expression builder for constructing <see cref="Freya{T}" /> computations.
-type FreyaBuilder () =
+/// Defines a computation expression builder for constructing <see cref="Core{T}" /> computations.
+type CoreBuilder () =
```

Comment:
Not unreasonable for sure, and I take your point about the naming. See other comments :)